xocto/ranges.py

```diff
@@ -832,3 +832,9 @@ def get_finite_datetime_ranges_from_timestamps(
         # or end of the period.
         if start != end
     ]
+
```

Comment:
🤔 I think this function might already exist as [`RangeSet.contains_range`](https://github.com/octoenergy/xocto/blob/main/xocto/ranges.py#L572-L574)?

Reply:
Yes somehow I missed that this did the same thing, but I checked now and you're right, thanks.